bvm/ballerina-runtime/src/main/java/module-info.java

```diff
@@ -48,7 +48,7 @@
     exports io.ballerina.runtime.internal.scheduling to io.ballerina.cli.utils, io.ballerina.java,
             io.ballerina.lang.array, io.ballerina.lang.error, io.ballerina.lang.internal, io.ballerina.lang.map,
             io.ballerina.lang.table, io.ballerina.lang.transaction, io.ballerina.lang.value, io.ballerina.lang.xml,
-            io.ballerina.log.api, io.ballerina.testerina.core, io.ballerina.testerina.runtime;
+            io.ballerina.log.api, io.ballerina.testerina.core, io.ballerina.testerina.runtime, io.ballerina.shell;
```

Comment:
I don't think we need to export this to shell module. Shall we remove it? As mentioned please use the `Runtime` util functions to invoke methods.